src/styles.scss

```diff
@@ -261,463 +650,6 @@ $error: #ff6939;
-  padding-right: 5px;
-}
-
-.p-2px {
-  padding: 2px;
-}
-.pt-2px {
-  padding-top: 2px;
-}
-.pr-2px {
-  padding-right: 2px;
-}
-.pb-2px {
-  padding-bottom: 2px;
-}
-.pl-2px {
-  padding-left: 5px;
-}
-.py-2px {
-  padding-top: 2px;
-  padding-bottom: 2px;
-}
-.px-2px {
-  padding-left: 2px;
-  padding-right: 2px;
-}
-
-// Zero padding classes are useful for things like bootstrap columns that have padding added.
-.p-0px {
-  padding: 0px;
-}
-.pt-0px {
-  padding-top: 0px;
-}
-.pr-0px {
-  padding-right: 0px;
-}
-.pb-0px {
-  padding-bottom: 0px;
-}
-.pl-0px {
-  padding-left: 0px;
-}
-.py-0px {
-  padding-top: 0px;
-  padding-bottom: 0px;
-}
-.px-0px {
-  padding-left: 0px;
-  padding-right: 0px;
-}
-
-// Custom margin classes.
-.mt-2px {
-  margin-top: 2px;
-}
-.mt-7px {
-  margin-top: 7px;
-}
-
-.m-5px {
-  margin: 5px;
-}
-.mt-5px {
-  margin-top: 5px;
-}
-.mr-5px {
-  margin-right: 5px;
-}
-.mb-5px {
-  margin-bottom: 5px;
-}
-.ml-5px {
-  margin-left: 5px;
-}
-
-.m-10px {
-  margin: 10px;
-}
-.mt-10px {
-  margin-top: 10px;
-}
-.mr-10px {
-  margin-right: 10px;
-}
-.mb-10px {
-  margin-bottom: 10px;
-}
-.ml-10px {
-  margin-left: 10px;
-}
-
-.m-12px {
-  margin: 12px;
-}
-.mt-12px {
-  margin-top: 12px;
-}
-.mr-12px {
-  margin-right: 12px;
-}
-.mb-12px {
-  margin-bottom: 12px;
-}
-.ml-12px {
-  margin-left: 12px;
-}
-
-.m-15px {
-  margin: 15px;
-}
-.mt-15px {
-  margin-top: 15px;
-}
-.mr-15px {
-  margin-right: 15px;
-}
-.mb-15px {
-  margin-bottom: 15px;
-}
-.ml-15px {
-  margin-left: 15px;
-}
-
-.m-16px {
-  margin: 16px;
-}
-.mt-16px {
-  margin-top: 16px;
-}
-.mr-16px {
-  margin-right: 16px;
-}
-.mb-16px {
-  margin-bottom: 16px;
-}
-.ml-16px {
-  margin-left: 16px;
-}
-.mx-16px {
-  margin-left: 16px;
-  margin-right: 16px;
-}
-
-.m-20px {
-  margin: 20px;
-}
-.mt-20px {
-  margin-top: 20px;
-}
-.mr-20px {
-  margin-right: 20px;
-}
-.mb-20px {
-  margin-bottom: 20px;
-}
-.ml-20px {
-  margin-left: 20px;
-}
-.mx-20px {
-  margin-left: 20px;
-  margin-right: 20px;
-}
-.my-20px {
-  margin-top: 20px;
-  margin-bottom: 20px;
-}
-
-.m-25px {
-  margin: 25px;
-}
-.mt-25px {
-  margin-top: 25px;
-}
-.mr-25px {
-  margin-right: 25px;
-}
-.mb-25px {
-  margin-bottom: 25px;
-}
-.ml-25px {
-  margin-left: 25px;
-}
-.mx-25px {
-  margin-left: 25px;
-  margin-right: 25px;
-}
-.my-25px {
-  margin-top: 25px;
-  margin-bottom: 25px;
-}
-
-.m-30px {
-  margin: 30px;
-}
-.mt-30px {
-  margin-top: 30px;
-}
-.mr-30px {
-  margin-right: 30px;
-}
-.mb-30px {
-  margin-bottom: 30px;
-}
-.ml-30px {
-  margin-left: 30px;
-}
-.mx-30px {
-  margin-left: 30px;
-  margin-right: 30px;
-}
-.my-30px {
-  margin-top: 30px;
-  margin-bottom: 30px;
-}
-
-.m-40px {
-  margin: 40px;
-}
-.mt-40px {
-  margin-top: 40px;
-}
-.mr-40px {
-  margin-right: 40px;
-}
-.mb-40px {
-  margin-bottom: 40px;
-}
-.ml-40px {
-  margin-left: 40px;
-}
-.mx-40px {
-  margin-left: 40px;
-  margin-right: 40px;
-}
-.my-40px {
-  margin-top: 40px;
-  margin-bottom: 40px;
-}
-
-.m-0auto {
```

Comment:
removing all of these is good, but we just gotta make sure we get rid of all uses as well.

Reply:
Yup, will do a cleanup past towards the end once I've gone through most of the screens & states.